constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -122,8 +132,12 @@ export class ClassEditPage extends React.Component {
             semesterId: this.state.selectSemesterId,
             locationId: this.state.selectLocationId,
             classKey: this.state.inputClassKey,
+            googleClassCode: this.state.googleClassCode,
+            paymentNotes: this.state.paymentNotes,
```

Comment:
move these fields below to be next to priceLump / pricePerSession.
Order hsould be fullState, googleClassCode, priceX, paymentNotes